Add Java's Calendar term computeFields (#3035)
Codecademy/docs: content/java/concepts/calendar/terms/computeFields/computeFields.md

---
Title: '.computeFields()'

> Description: 'Converts the current millisecond time value time to calendar field values in fields[].'

Reviewer: This text is an exact copy of the oracle documentation. That is considered plagiarism. All docs content must be original. Please revise the content with your own words/definition and make any relevant changes to line 15, so that the explanations are in alignment.

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Calendar'
  - 'Methods'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **`.computeFields()`** method of the `Calendar` class converts the current millisecond time value to the relevant field values in the passed `Calendar` object.

Reviewer 2: ```suggestion
The **`.computeFields()`** method of the `Calendar` class converts the current millisecond time value to the relevant field values in the `Calendar` object.
```

## Syntax

```pseudo
calendar.computeFields()
```

> > **Note:** `computeFields()` method is a non-static method, it is accessible with the `class` object. Trying to access it with the `class` name throws an error.

Reviewer 2: ```suggestion
> **Note:** `computeFields()` method is a non-static method, it is accessible with the `Calendar` instance. Trying to access it with the `class` name throws an error.
```

## Example

> In the example below the `CalendarComputeFieldExample` class is instantiated and the calendar is set to the year `2024`. Then, the `computeFields()` method is called to sync up the `calendar` field values with the new time that has been set for the `calendar` object.

Reviewer 2: ```suggestion
In the example below the `CalendarComputeFieldExample` class is instantiated and the calendar is set to the year `2024`. Then, the `computeFields()` method is called to sync up the `Calendar` field values with the new time that has been set for the `Calendar` object.
```

```java

> import java.util.*;

Reviewer: The code should be indented with 2 spaces, tabs are not used in docs. Also update all comments to begin with a capital

Author: @caupolicandiaz  I fixed the indentation. What do you intend by "update comments"? What comments?

Reviewer: The comments within the code should look like this -

// This is correct

instead of 

//this is not the right format
